### PR TITLE
 Ru: translation of simple-profiling.md (II part)

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
@vasbaz, Maybe you should remove the last empty space for some sentences ;)
PS: You can use `npm run test` locally and resubmit your fixed changes.

**Comment by ghost** [comment]
@nodejs/nodejs-ru, Please have a review of his translation except for the formation problem, thanks!

File: locale/ru/docs/guides/simple-profiling.md [hunk 1]
```diff
@@ -123,27 +123,27 @@ Percentage of the requests served within a certain time (ms)
...
 ```
 
-From this output, we see that we're only managing to serve about 5 requests per
-second and that the average request takes just under 4 seconds round trip. In a
-real world example, we could be doing lots of work in many functions on behalf
-of a user request but even in our simple example, time could be lost compiling
-regular expressions, generating random salts, generating unique hashes from user
-passwords, or inside the Express framework itself.
+По выводу видно, что обрабатывается только около 5 запросов в секунду, а средний 
+запрос занимает чуть менее 4 секунд в оба конца. В реальной ситуации могло бы
```
Comment (alexandrtovmach) [inline locale/ru/docs/guides/simple-profiling.md]: не "средний запрос", а "запрос в среднем"

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]

**Comment by XhmikosR** [comment]
@vasbaz can you take care of @alexandrtovmach's comment and fix tests?

File: locale/ru/docs/guides/simple-profiling.md [hunk 2]
```diff
@@ -123,27 +123,27 @@ Percentage of the requests served within a certain time (ms)
  100%   4225 (longest request)
 ```
 
-From this output, we see that we're only managing to serve about 5 requests per
-second and that the average request takes just under 4 seconds round trip. In a
-real world example, we could be doing lots of work in many functions on behalf
-of a user request but even in our simple example, time could be lost compiling
-regular expressions, generating random salts, generating unique hashes from user
-passwords, or inside the Express framework itself.
+По выводу видно, что обрабатывается только около 5 запросов в секунду, а средний
```
Comment (Aissaoui-Ahmed) [inline locale/ru/docs/guides/simple-profiling.md]:
```suggestion
По выводу видно, что обрабатывается только около 5 запросов в секунду, а средний
```

File: locale/ru/docs/guides/simple-profiling.md (hunk 1 above)
Comment (Aissaoui-Ahmed) [inline locale/ru/docs/guides/simple-profiling.md]:
```suggestion
запрос занимает чуть менее 4 секунд в оба конца. В реальной ситуации могло бы
```

**Comment by Aissaoui-Ahmed** [comment]
Remove the last spaces on the lines `126...130,133,134,145`

See : [CI -> Run tests ](https://github.com/nodejs/nodejs.org/pull/2579/checks?check_run_id=253277194)

```bash
[htmllint] found 0 errors out of 30 files
locale/ru/docs/guides/simple-profiling.md: 126: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
locale/ru/docs/guides/simple-profiling.md: 127: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
locale/ru/docs/guides/simple-profiling.md: 128: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
locale/ru/docs/guides/simple-profiling.md: 129: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
locale/ru/docs/guides/simple-profiling.md: 130: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
locale/ru/docs/guides/simple-profiling.md: 133: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
locale/ru/docs/guides/simple-profiling.md: 134: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
locale/ru/docs/guides/simple-profiling.md: 145: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
```

**Comment by alexandrtovmach** [comment]
@XhmikosR @Aissaoui-Ahmed 
I think we need to close this PR or add **staled** label.

**Comment by XhmikosR** [comment]
If you agree with the other changes, you can just apply the fixes and move forward.

**Comment by alexandrtovmach** [comment]
I don't any changes here, just requested changes =)

**Comment by XhmikosR** [comment]
@alexandrtovmach I mean this one

>не "средний запрос", а "запрос в среднем"

I don't speak Russian, but if you want this change just edit the file :P

**Comment by alexandrtovmach** [comment]
@XhmikosR Yeah, but also need to update lint issues... and here just a part of file translated

File: locale/ru/docs/guides/simple-profiling.md [hunk 3]
```diff
@@ -123,27 +123,27 @@ Percentage of the requests served within a certain time (ms)
...
```
Comment (alexandrtovmach) [inline locale/ru/docs/guides/simple-profiling.md:126]:
```suggestion
По выводу видно, что обрабатывается только около 5 запросов в секунду, а запрос
```

File: locale/ru/docs/guides/simple-profiling.md [hunk 4]
```diff
@@ -123,27 +123,27 @@ Percentage of the requests served within a certain time (ms)
...
+запрос занимает чуть менее 4 секунд в оба конца. В реальной ситуации могло бы
```
Comment (alexandrtovmach) [inline locale/ru/docs/guides/simple-profiling.md:127]:
```suggestion
в среднем занимает чуть менее 4 секунд в оба конца. В реальной ситуации могло бы
```

File: locale/ru/docs/guides/simple-profiling.md [hunk 5]
```diff
@@ -277,5 +277,5 @@ Hopefully, through the performance investigation of this (admittedly contrived)
 example, you've seen how the V8 tick processor can help you gain a better
 understanding of the performance of your Node.js applications.
 
-[profiler inside V8]: https://v8.dev/docs/profile
+[профайлер V8]: https://v8.dev/docs/profile
```
Comment (alexandrtovmach) [inline locale/ru/docs/guides/simple-profiling.md:280]: потому что ссылка не будет работать
```suggestion
[profiler inside V8]: https://v8.dev/docs/profile
```

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]

**Comment by XhmikosR** [comment]
That's OK I trimmed the whitespaces.

**Review by alexandrtovmach — APPROVED** [review]

**Comment by alexandrtovmach** [comment]
I'll update file by my own

**Pull request merged**